Algorithm.Python/build.bat

```diff
@@ -6,10 +6,10 @@ SET pyc="C:\Program Files (x86)\IronPython 2.7\Tools\Scripts\pyc.py"
 
 REM Clean the output directory
 del "QuantConnect.Algorithm.Python.dll"
-del "../../../Engine/bin/Debug/QuantConnect.Algorithm.Python.dll"
+del "../Launcher/bin/Debug/QuantConnect.Algorithm.Python.dll"
 
 REM IronPython Compile the Algorithm
 ipy %pyc% /target:dll /out:QuantConnect.Algorithm.Python main.py
 
 REM Copy to the Lean Algorithm Project
-echo f|xcopy /Y "QuantConnect.Algorithm.Python.dll" "../../../Engine/bin/Debug/QuantConnect.Algorithm.Python.dll"
\ No newline at end of file
+echo f|xcopy /Y "QuantConnect.Algorithm.Python.dll" "../Launcher/bin/Debug/QuantConnect.Algorithm.Python.dll"
```

Comment:
Yes! Working on it.
I need to be able to run Lean on Linux to be sure the build.sh is done right.